### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by topugit**

### CODE REVIEW

File: algorithms/001514-path-with-maximum-probability/README.md
Comment (topugit) [inline algorithms/001514-path-with-maximum-probability/README.md:1]: done

File: algorithms/001514-path-with-maximum-probability/solution.php
Comment (topugit) [inline algorithms/001514-path-with-maximum-probability/solution.php:1]: done

**Review by topugit — COMMENTED** [review]
done

**Comment by mah-shamim** [comment]
pull request merged done

**Pull request merged**